src/components/Camera/index.js

```diff
@@ -76,14 +76,13 @@ type CameraType = {
   ...CameraCommonType,
   onScreenshot: Function,
   trackScreen: Function,
-
 }
 
 export default class Camera extends React.Component<CameraType> {
 
-  webcam: ?Webcam = null
+  webcam: Webcam = null
   interval: ?Visibility
-  countdown: ?CountUp
+  countdown = null
```

Comment:
Why do we need this?

Reply:
I think is so that Flow picks up that this is a property and it can be null